adminapi/request.py

```diff
@@ -128,7 +128,9 @@ def json_encode_extra(obj):
     if isinstance(obj, BaseFilter):
         return obj.serialize()
     if isinstance(obj, datetime):
-        return obj.strftime('%Y-%m-%d %H:%M:%S')
+        if obj.tzinfo is None:
+            obj = obj.replace(tzinfo=timezone.utc)
+        return obj.astimezone(timezone.utc).strftime('%Y-%m-%d %H:%M:%S%z')
```

Comment:
Why do you need this return.  Just casting it to string later would be fine, no?

Reply:
No. Casting to string would introduce milliseconds in the time and a colon in the utc offset like +01:00 instead of +0100. Pythons buildin strptime parsing is pretty rigid and doesn't allow for optional fields. We couldn't parse the result on the receiving side:

```python
> datetime.strptime('2019-03-23 18:29:08.374537+00:00', '%Y-%m-%d %H:%M:%S%z')

ValueError: time data '2019-03-23 18:29:08.374537+00:00' does not match format '%Y-%m-%d %H:%M:%S%z'
```